NuclearData/mgNeutronData/baseMgNeutron/baseMgNeutronDatabase_class.f90

```diff
@@ -45,7 +45,7 @@ module baseMgNeutronDatabase_class
   !!   mats       -> array containing all defined materials (by matIdx)
   !!   majorant   -> majorant xs for delta tracking
   !!   activeMats -> list of matIdxs of materials active in the problem
-  !!   nG         -> number of energy groups
+  !!   nG         -> number of energy groups (from superclass)
```

Comment:
```suggestion
```
Maybe we should move this to `mgNeutronDatabase`. (add there `Public Members:` section with nG?